Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Stage 0 : Build the C library 
+
+FROM debian:buster-20190506 AS lib_builder
```

Comment:
We were lucky that this version uses glibc 2.28 but we need to find a good map from the debian version to the glibc version. This will matter when we upgrade the rpi OS.